From: Keith Packard <keithp@keithp.com>
Date: Mon, 4 Oct 2021 09:33:58 -0700
Subject: Remove dependencies that may no be found at runtime

Some distributions are not found by pkg_resources when load_entry_point is
called at startup
---
 setup.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/setup.py b/setup.py
index 0861243..f0dd424 100644
--- a/setup.py
+++ b/setup.py
@@ -76,6 +76,12 @@ install_requires = [
     "pywin32; sys_platform=='win32'",
 ]
 
+exclude = ("PyQt5", "QScintilla", "qtconsole", "PyQtChart")
+install_requires = [
+    requirement
+    for requirement in install_requires
+    if not requirement.startswith(exclude)
+]
 
 extras_require = {
     "tests": [
